backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -157,7 +157,7 @@
   </table>
 <% else %>
   <div class="alpha twelve columns no-objects-found">
-    <%= Spree.t(:no_resource_found, resource: I18n.t(:other, scope: 'activerecord.models.spree/order')) %>,
+    <%= Spree.t(:no_resource_found, resource: Spree::Order.model_name.human(count: :other))  %>,
```

Comment:
Every time I see this in our shops, translated to a German sentence, I start shrugging. 

This is maybe out of scope of this PR, but we need to address this one. Just wanted to point the attention to this one here.